Blueprints/BlueprintDefinitions/vs2022/AspNetCoreWebAPI-Image/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -52,7 +52,7 @@ have the .NET project built inside the container. Below is an example of buildin
 
 ```dockerfile
 FROM public.ecr.aws/lambda/dotnet:8
-FROM mcr.microsoft.com/dotnet/sdk:8.0-bullseye-slim as build
+FROM mcr.microsoft.com/dotnet/sdk:8.0-bookworm-slim as build
```

Comment:
In all of the templates I changed this to be just `mcr.microsoft.com/dotnet/sdk:8.0` and missed this one. Can we use `mcr.microsoft.com/dotnet/sdk:8.0` here?

Reply:
Sure thing, fixed in [4b33302](https://github.com/aws/aws-lambda-dotnet/pull/1677/commits/4b333020f9b8bb49f62f10b28ada2fd0071ba4d9)